plugins/org.eclipse.payara.tools/src/org/eclipse/payara/tools/sdk/server/JDK.java

```diff
@@ -25,23 +25,25 @@
 
 package org.eclipse.payara.tools.sdk.server;
 
+import java.util.Arrays;
 import java.util.Optional;
 
 /**
- * A simple class that fills a hole in the JDK.  It parses out the version numbers
- * of the JDK we are running.
- * 
+ * A simple class that fills a hole in the JDK. It parses out the version
+ * numbers of the JDK we are running.
+ *
  * <p>
  * Example: <br>
- * 
+ *
  * 1.6.0_u14 == major = 1 minor = 6, subminor = 0, update = 14
  *
  * @author bnevins
  */
 public final class JDK {
```

Comment:
There's a lot of intricate logic in this class. I would suggest some unit tests

Reply:
Created the unit test for JDK version parsing but eclipse plugin build is not picking it.

Reply:
@dmatej @MarkWareham help needed on this.

Reply:
Run As -> JUnit Plug-in Test?

Reply:
Eh, but the class is not a plugin ... hmm. I'm not sure how to run unit tests inside eclipse extension project ...

Reply:
Created ECOSYS-88 to address the ant build issue.